langchain/agents/conversational_chat/base.py

```diff
@@ -85,18 +92,17 @@ def create_prompt(
         )
         tool_names = ", ".join([tool.name for tool in tools])
         _output_parser = output_parser or AgentOutputParser()
-        format_instructions = human_message.format(
+        prompt = system_message + "\n\n" + tool_message
+        # Format in two steps because the format instructions includes tool_names
+        final_prompt = prompt.format(
             format_instructions=_output_parser.get_format_instructions()
-        )
-        final_prompt = format_instructions.format(
-            tool_names=tool_names, tools=tool_strings
-        )
+        ).format(tool_names=tool_names, tools=tool_strings)
         if input_variables is None:
             input_variables = ["input", "chat_history", "agent_scratchpad"]
         messages = [
-            SystemMessagePromptTemplate.from_template(system_message),
+            SystemMessagePromptTemplate.from_template(final_prompt),
```

Comment:
@hwchase17 was this intentionally inserted as human messages due to this?
```
[gpt-3.5-turbo-0301](https://platform.openai.com/docs/models) does not always pay strong attention to system messages. Future models will be trained to pay stronger attention to system messages
```

(from https://platform.openai.com/docs/guides/chat/introduction)

We may want to restructure the default `Prompt` as well to account for the change, since it's structured as the Human giving the tool responses.

We ought to share an evaluation of the behavior before and after the change before landing this

Reply:
In my experience, gpt-3 also doesn't pay strong attention to user messages. :) In fact, the rationale for moving it was that hopefully for future models it would be better.

How would you do the evaluation? I honestly don't really know how to test these sorts of changes.